test/tools/spec-runner/index.js

```diff
@@ -699,13 +702,11 @@ const kOperations = new Map([
   ],
   [
     'mapReduce',
+    // eslint-disable-next-line no-unused-vars
     (operation, collection, context /*, options */) => {
-      const args = operation.arguments;
-      const map = args.map;
-      const reduce = args.reduce;
-      const options = { session: maybeSession(operation, context) };
-      if (args.out) options.out = args.out;
-      return collection.mapReduce(map, reduce, options);
+      throw new Error(
```

Comment:
The mapReduce retryable-reads tests are somehow skipping so this error didn't make those tests fail, I'm not seeing it in the filter in `retryable_reads.spec.test.js`, any ideas?

Reply:
Yeah, this threw me for a loop initially too.  The `filter` argument in `generateTopologyTests` returns true for tests that run, false otherwise.  The `filter` for the retryable reads spec test is actually an inclusion filter, not exclusion.

I added the doc comment to `generateTopologyTests` after getting confused by that initially.

Reply:
I considered a minor refactor but decided not to:

```typescript
  generateTopologyTests(testSuites, testContext, spec => {
    const testsToRun = [
      ... list all test regexes here
    ];
    return testsToRun.some(regex => spec.description.match(regex));
  });

```

want me to make the change?

Reply:
Your call, I'm happy that we looked into it and know why they weren't run